### PR TITLE
Split vars

### PR DESCRIPTION
**Pull request by julialongtin**
move version info into something to be supplied to this repo, rather than part of the repo. round 1 of N.

### CODE REVIEW

File: vpc/ansible/populate_github_repos-tasks.yml [hunk 1]
```diff
@@ -17,36 +17,17 @@
...
-  loop:
-   - 'wireapp'
-   - 'elastic'
-   - 'ANXS'
-   - 'geerlingguy'
-   - 'githubixx'
-   - 'andrewrothstein'
-   - 'cchurch'
-   - 'kubernetes-sigs'
+  with_items: '{{ github_repos }}'
```
Comment (lucendio) [inline vpc/ansible/populate_github_repos-tasks.yml:28]: GH doesn't let me to suggest.

For consistency: ~`with_items`~ --> `loop: '{{ github_repos }}'`

File: vpc/ansible/vars_examples/github_repos.yml [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+  - 'ANXS/apt.git'
+  - 'geerlingguy/ansible-role-java.git'
+  - 'geerlingguy/ansible-role-ntp.git'
+  - 'githubixx/ansible-role-kubectl.git'
+  - 'andrewrothstein/ansible-kubernetes-helm.git'
+  - 'cchurch/ansible-role-admin-users.git'
+  - 'kubernetes-sigs/kubespray.git'
+  - 'nickhammond/ansible-logrotate.git'
+
+
```
Comment (lucendio) [inline vpc/ansible/vars_examples/github_repos.yml:20]:
```suggestion
  - 'nickhammond/ansible-logrotate.git'

```

**Review by lucendio — CHANGES_REQUESTED** [review]
*After a clarifying conversation*

File: vpc/ansible/populate_github_repos-tasks.yml (hunk 1 above)
Comment (julialongtin) [inline vpc/ansible/populate_github_repos-tasks.yml:28]:
```suggestion
  loop: '{{ github_repos }}'
```

**Pull request merged**